LICENSE

```diff
@@ -1,202 +0,0 @@
-
```

Comment:
put back

Reply:
@julianlitz I think you still removed the license? can you restore it?

Reply:
The LICENSE file was restored on the 21st of October.